apps/explorer/lib/explorer/chain.ex

```diff
@@ -1275,8 +1275,8 @@ defmodule Explorer.Chain do
   Checks to see if the chain is down indexing based on the transaction from the
   oldest block and the pending operation
   """
-  @spec finished_internal_transactions_indexing?([api?]) :: boolean()
-  def finished_internal_transactions_indexing?(options \\ []) do
+  @spec finished_indexing_internal_transactions?([api?]) :: boolean()
+  def finished_indexing_internal_transactions?(options \\ []) do
     internal_transactions_disabled? = System.get_env("INDEXER_DISABLE_INTERNAL_TRANSACTIONS_FETCHER", "false") == "true"
```

Comment:
What if we will do like this? Just in case if in future we will need to use this function somewhere else
```suggestion
    internal_transactions_disabled? = System.get_env("INDEXER_DISABLE_INTERNAL_TRANSACTIONS_FETCHER", "false") == "true" or not Application.get_env(:indexer, Indexer.Supervisor)[:enabled]
```

Reply:
I also suggest to use `Application.get_env(:indexer, Indexer.Fetcher.InternalTransaction.Supervisor)[:disabled?]` instead of `System.get_env("INDEXER_DISABLE_INTERNAL_TRANSACTIONS_FETCHER", "false")`

Reply:
Makes sense. @nikitosing @sl1depengwyn both comments processed in https://github.com/blockscout/blockscout/pull/7096/commits/26d80e0245ccecbdcec6ab9fb43b9372e553d896